merge_tag_release.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+
+OptionParser.new do |opts|
+  opts.banner = "Usage: merge_tag_release.rb [options]"
+
+  opts.on("-c", "-commit [COMMIT]", "SHA hash of the commit to be cherry-picked") do |project|
+    options[:project] = project
+  end
+
+  opts.on("-b", "-branches [BRANCHES]", "Branches to merge commit to, separated by commas. e.g: '1.16-stable, 1.15-stable' ") do |branches|
+    options[:branches] = branches
+  end
+
+  opts.on("-p", "-project [PROJECT]", "Name of the project e.g: rubygem-foreman_discovery") do |project|
+    options[:project] = project
+  end
+end.parse!
+
+raise OptionParser::MissingArgument, "--branches" if options[:branches].nil?
+raise OptionParser::MissingArgument, "--project" if options[:project].nil?
+raise OptionParser::MissingArgument, "--commit" if options[:commit].nil?
+CORE_PROJECTS = %q(foreman foreman-selinux foreman-proxy foreman-installer)
+
+branches = options[:branches].split(',').map!(&:strip)
+
+branches.each do |branch|
+  `git checkout rpm/#{branch}`
+  `git cherry-pick #{options[:commit]}`
+  # Cherry-pick commit to branch
+  `tito tag --keep-version`
+  `git push origin rpm/#{branch} --tags`
```

Comment:
As I recall ```--tags``` only pushes tags, not the actual branches. I also don't have an ```origin``` remote but always use ```ekohl``` and ```upstream```. Another thing is that I personally don't push changes to my personal remotes, just the branches for PRs.